### ISSUE TITLE
Feature request: highlight lines in the code

### ISSUE THREAD

**Issue by DanRoscigno** [christian-bromann/docusaurus-theme-github-codeblock#137]
I tried setting the label on this issue to feature, but I do not have permission to add labels.

With the default Docusaurus codeblocks I am able to add comments to codeblocks
to indicate the start and stop of highlighted sections:

```sql
-- Create table from S3 using FILES() table function
CREATE TABLE DocsQA.user_behavior_inferred
AS SELECT * FROM FILES (
	"path" = "s3://starrocks-examples/user_behavior_ten_million_rows.parquet",
	"format" = "parquet",
	"aws.s3.region" = "us-east-1",
	-- highlight-start
	"aws.s3.access_key" = "AAAAAAAAAAAAAAAAAAAA",
	"aws.s3.secret_key" = "BBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBB"
	-- highlight-end
);
```

renders as:

<img width="957" alt="image" src="https://github.com/christian-bromann/docusaurus-theme-github-codeblock/assets/25182304/3b46665a-6e4f-49f3-b5a8-51a53f253859">

Your code (which is wonderful and very important to me) seems to be noticing that the comments are meant to be processed but not displayed, as  I do not see the comments when rendered using your package:
<img width="952" alt="image" src="https://github.com/christian-bromann/docusaurus-theme-github-codeblock/assets/25182304/d18f5234-8c4b-4bbd-938f-5994acc73d59">

**Comment by DanRoscigno** [comment]
Ah! I apologize, it works fine. I forgot that I included the hash in the URL that I was referencing. I had added the highlighting comments in a newer commit.  I just updated and everything is beautiful.

My use case may interest you, I am instituting a testing procedure for the documentation for the StarRocks database. I want to source code snippets from end to end tests so that I have warning when I need to update the documentation because of a breaking change.

Thanks again!
